simple-facebook-instant-articles.php

```diff
@@ -736,6 +737,26 @@ protected function get_simple_reach_analytics_code() {
 	}
 
 	/**
+	 * Get Chartbeat script in the FB IA format.
+	 *
+	 * Ref: https://developers.facebook.com/docs/instant-articles/reference/analytics
+	 *
+	 * @return string Chartbeat script in FB IA format.
+	 */
+	protected function get_chartbeat_analytics_code() {
+
+		if (
+			lawrence_option_enabled( 'usat_chartbeat_enabled' )
+			&& function_exists( 'usat_chartbeat_add_header' )
+			&& function_exists( 'usat_chartbeat_add_footer' )
+		) {
+			ob_start();
+			require( trailingslashit( $this->template_path ) . 'script-chartbeat.php' );
+			return ob_get_clean();
```

Comment:
Let's have it as a separate PR? I think this one is ok to merge if you're happy with it